src/app/settings/keyboard-mapping-settings/components/keyboard-mapping-settings-page/keyboard-mapping-settings-page.component.ts

```diff
@@ -227,6 +240,14 @@ export class KeyboardMappingSettingsPageComponent implements OnInit, OnDestroy {
     )
   }
 
+  public getKeyboardMappingActionContentType(keyboardMapping: KeyboardMapping): Tv2ActionContentType {
+    return keyboardMapping.action ? keyboardMapping.action.metadata.contentType : Tv2ActionContentType.UNKNOWN
+  }
+
+  public getKeyboardMappingActionName(keyboardMapping: KeyboardMapping): string {
+    return keyboardMapping.action ? keyboardMapping.action.name : ''
+  }
```

Comment:
You should be able to use the double coalescing operator here instead of the ternary:
![image](https://github.com/tv2/ng-sofie/assets/23049647/4eef3ff8-9e50-4a1b-afeb-edde6798cfd4)